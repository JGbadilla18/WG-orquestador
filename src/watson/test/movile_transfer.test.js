const testWatson = require('./testWatson');

watsonText = 'Quiero hacer una transferencia movil.'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
