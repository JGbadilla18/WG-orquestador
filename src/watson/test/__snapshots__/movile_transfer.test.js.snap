// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quiero hacer una transferencia movil. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "buttonType": "web_url",
      "buttons": Array [
        Object {
          "title": "Conoce más aquí",
          "url": "https://www.corporacionbi.com/gt/bancoindustrial/transferencias-moviles-5071",
        },
      ],
      "response_index": 0,
      "response_type": "button",
      "text": "Transferencias Móviles es el servicio gratuito que te permite enviar dinero en efectivo a cualquier persona que tenga un celular sin necesidad que tenga una Cuenta Bi.",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
